workspace/busStop.ts

```diff
@@ -0,0 +1,31 @@
+const request = require('request')
+import {Bus} from './bus'
+import * as _ from "lodash"
+
+export class BusStop {
+    constructor(public id: string, public name: string) {
+    }
+    getBuses(): Promise <Bus[]> {
+        let promiseLatAndLong: Promise <string> = new Promise((resolve) => {
+        var requestString = "https://api.tfl.gov.uk/StopPoint/" + 
+            this.id + "/Arrivals?app_id=ef0cca24&app_key=281fde5472478ee61c2d8a63c5e4f554"
```

Comment:
I like that you've separated out the requestString into a separate variable - cleans up the request call below! (missing a semicolon, though...